package.json

```diff
@@ -18,14 +18,16 @@
     "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-react": "7.32.2",
     "husky": "8.0.3",
-    "isomorphic-fetch": "3.0.0",
     "jsdom": "22.1.0",
     "lerna": "7.1.3",
     "npm-run-all": "4.1.5",
     "prettier": "3.0.0",
     "rxjs-compat": "6.6.7",
     "typescript": "5.1.6"
   },
+  "dependencies": {
+    "isomorphic-fetch": "3.0.0"
+  },
```

Comment:
Was that needed to be moved from ```devDependencies``` to ```dependencies```?

Reply:
after changing [scripts/get-changelog.js](https://github.com/jaegertracing/jaeger-ui/pull/1616/files#diff-106ea5da4aa3b89f6740639ff95dc458220eed23cde798e70c95571e8a30e8e7) there was an error ('isomorphic-fetch' should be listed in the project's dependencies, not devDependencies). So I modified it and it worked fine.